src/http_server.h

```diff
@@ -196,6 +206,12 @@ class HTTPAPIServer : public HTTPServer {
       const RestrictedFeatures& restricted_apis,
       std::unique_ptr<HTTPServer>* http_server);
 
+
+  static bool Create_Wrapper(
```

Comment:
Would replace `Create_Wrapper` with `CreateWrapper` everywhere for style consistency

Reply:
Done.